server.js

```diff
@@ -21,7 +21,7 @@ server.get('/home*', allowIfLoggedOut, (req, res) => {
     res.render('home');
 });
 
-server.get('/dash', allowIfLoggedIn, (req, res) => {
+server.get('/dash*', allowIfLoggedIn, (req, res) => {
```

Comment:
Why do we need this?

Reply:
This allows routes other than just dash like /dash/about or else it gives an error 'cannot GET ___'